dashboard/app/models/lti_integration.rb

```diff
@@ -22,11 +22,16 @@
 #  index_lti_integrations_on_platform_id  (platform_id)
 #
 class LtiIntegration < ApplicationRecord
-  validates :platform_id, presence: true
   validates :issuer, presence: true
   validates :client_id, presence: true
   validates :platform_name, presence: true
   validates :auth_redirect_url, presence: true
   validates :jwks_url, presence: true
   validates :access_token_url, presence: true
+
```

Comment:
Should we enforce uniqueness on the uuid? I know it's incredibly unlikely to get duplicates, but it's also easy to enforce here and in the table.

```suggestion
validates :uuid, uniqueness: true

```

Reply:
Yep, incredibly unlikely. However since it's easy to enforce I added it in.